redis/src/test/resources/start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Deduce whether we are running cygwin
+case $(uname -s) in
+(CYGWIN*) cygwin=true;;
+(*) cygwin=;;
+esac
+
+root=$(cd "$(dirname "$(readlink $0)")"; pwd)
+
+exec gradle test --tests org.apache.calcite.adapter.redis.RedisServerMini -x test
+:compileJava UP-TO-DATE
+:processResources UP-TO-DATE
+:classes UP-TO-DATE
+:jar UP-TO-DATE
+:assemble UP-TO-DATE
+:check
+:build
```

Comment:
is it intentional?

Reply:
> is it intentional?

yes

Reply:
What is the purpose?

Reply:
Let me modify it, too many parameters are specified. It seems that this start.sh startup script is not needed, because the service will be started when Junit-test is compiled and executed.